docs/mp/tracing.adoc

```diff
@@ -39,6 +39,9 @@ include::{rootdir}/includes/mp.adoc[]
 
 == Overview
 
+WARNING: MicroProfile Tracing specification is discontinued. This feature is marked as `@Deprecated` in
```

Comment:
This is not correct. MicroProfile OpenTracing (that is the spec name) is still there.
The OpenTracing specifciation that MP OpenTracing is based on is no longer maintained. That is the reason we are deprecating this feature.
The MP OpenTracing specification is no longer required by MicroProfile, and we (as in Helidon) have decided to deprecate it

Reply:
Redefined